src/services/groups.ts

```diff
@@ -38,16 +38,29 @@ interface GroupResponse {
   isPublic: boolean;
 }
 
-export async function groupsByRegion(region: number): Promise<GroupResponse[]> {
+export async function groupsByRegion(region: number, userId?: number): Promise<GroupResponse[]> {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let where: any = {
```

Comment:
```suggestion
  let where: WhereOptions = {
```

Reply:
also change line to import { Op, WhereOptions } from 'sequelize';

Reply:
(please, sorry)

Reply:
done.